### PR TITLE
Add more tables to db/seeds/dev.rb [#274]

### LINKED ISSUE #274: Add more tables to db/seeds/dev.rb
#261 seeks development seed data
#273 adds some seed data

Add seed data for dev for additional tables like `Requester` and `BoxRequest`

### PR DESCRIPTION
**Pull request by matisnape**
Resolves #274 

### Description

Added seeds for Requester and BoxRequest

### Type of change

- New feature (non-breaking change which adds functionality)

I also fixed missing end and formatted the app/models/box.rb file

### How Has This Been Tested?

I ran the rake task locally and verified in view

### CODE REVIEW

**Comment by maebeale** [comment]
@matisnape hello! thank you for this PR! 

would you be willing to merge the latest from develop into your branch?

**Comment by matisnape** [comment]
@maebeale done! :)

**Review by maebeale — COMMENTED** [review]
Hey there! Thanks for doing this merge! I think the research email needs to be outdented and  the assembly one is now in there twice?

**Comment by matisnape** [comment]
@maebeale True! Also, there was another missing done, I guess. 
Done :)

**Review by maebeale — APPROVED** [review]
Thank you, @matisnape ! Normally changes to indentation would be in their own PR, but this is fine. We needed to do what you did here anyway, so will accept it as part of this. Great job!

File: db/seeds/dev.rb [hunk 1]
```diff
@@ -24,3 +25,58 @@
     CoreBoxItem.joins(:abuse_type, :inventory_type).where(abuse_type: abuse_type, inventory_type: inventory_type).first_or_create!
   end
 end
+
+40.times do
+  is_underage = [{boolean: false, value: "12+"}, { boolean: true, value: "UNDERAGE"}].sample
+
+  requester = Requester.create!(
```
Comment (z3ugma) [inline db/seeds/dev.rb:32]: Hello! I have an unsolicited suggestion :) 
You can use the Factory from `spec/factories/requester.rb`, that way all the faker logic only has to live in one place if it needs updates

File: db/seeds/dev.rb [hunk 2]
```diff
@@ -24,3 +25,58 @@
...
+    county: Faker::Address.country,
+    phone: [Faker::Number.number(3), Faker::Number.number(3), Faker::Number.number(4)].join('-'),
+    ok_to_email: [true, false].sample,
+    ok_to_text: [true, false].sample,
+    ok_to_call: [true, false].sample,
+    ok_to_mail: [true, false].sample,
+    underage: is_underage[:boolean]
+  )
+
+  is_safe = [{boolean: false, value: "NOT SAFE"}, { boolean: true, value: "safe"}].sample
```
Comment (z3ugma) [inline db/seeds/dev.rb:50]: Same goes for `spec/factories/box_request.rb`

File: db/seeds/dev.rb (hunk 1 above)
Comment (matisnape) [reply db/seeds/dev.rb:32]: Yeah, I thought about that after I created a PR. 
I will work on this and the original issue (seeds for other tables) later this month :)

**Pull request merged**